deeppavlov/skills/dsl_skill/handlers/faq_handler.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import sqlite3
+from collections import defaultdict
+from typing import Callable, Optional, DefaultDict, Any
+
+from deeppavlov import train_model
+from deeppavlov.skills.dsl_skill.context import UserContext
+from deeppavlov.skills.dsl_skill.utils import execute_query
+from .handler import Handler
+
+
+class FAQStorage:
+    """
+    Helper class, that works with database.
+
+    Args
+        handler_name: name of handler function
+
+    """
+
+    def __init__(self, handler_name: str, db_name: str = "faq.db"):
+        self.handler_name = handler_name
+        self.connection_string = db_name
+        self.__connect()
+        execute_query(self.connection_string, "CREATE TABLE IF NOT EXISTS handler_intent("
```

Comment:
Why do you `self.__connect()` if the resulting connection is never used? Or why do you use `self.connection_string` for queries if you already have `self.connection`?